### PR TITLE
Changing loader flow

### PR DESCRIPTION
**Pull request by FedericoChavazza**
## Links:

https://trello.com/c/q6r3c0QU/9-show-loader-when-changing-movies-tabs

## What & Why:

now when movies arent in the carousel, shows loading gif

## Risks, Mitigation & Rollback:

- [x] Safe to Revert *check this box if this PR can be reverted without incident*

### CODE REVIEW

File: src/pages/MovieInfo.js [hunk 1]
```diff
@@ -13,10 +13,10 @@ import { BsFillBookmarkFill } from 'react-icons/bs';
 import { useState } from 'react';
 import Modal from 'components/modalComponent/Modal';
 import { Slider } from 'components/viewComponents/Slider';
+
```
Comment (JulianPasquale) [inline src/pages/MovieInfo.js]: Let's avoid changing this file. Doesn't seem to be related to this PR

**Review by JulianPasquale — APPROVED** [review]

**Review by martoesses — APPROVED** [review]
